api/src/main/java/org/openmrs/module/labintegration/api/hl7/messages/gnerators/pv1/Pv1AssignedPatientLocationHelper.java

```diff
@@ -51,9 +51,9 @@ public void assignedAlternativeVisitId(PV1 pv1, HL7Config hl7Config, Encounter e
 		}
 
 		try {
-			PatientIdentifier id = PatientUtil
-					.getPatientIdentifier(encounter.getPatient(), ECID_UUID);
-			pv1.getAlternateVisitID().getIDNumber().setValue(id.getIdentifier());
+			//PatientIdentifier id = PatientUtil.getPatientIdentifier(encounter.getPatient(), ECID_UUID);
```

Comment:
This error pops up on the CI:
```
/home/runner/work/openmrs-module-labintegration/openmrs-module-labintegration/api/src/main/java/org/openmrs/module/labintegration/api/hl7/messages/gnerators/pv1/Pv1AssignedPatientLocationHelper.java:8:8: error: Unused import - org.openmrs.PatientIdentifier.
/home/runner/work/openmrs-module-labintegration/openmrs-module-labintegration/api/src/main/java/org/openmrs/module/labintegration/api/hl7/messages/gnerators/pv1/Pv1AssignedPatientLocationHelper.java:11:8: error: Unused import - org.openmrs.module.labintegration.api.hl7.messages.util.PatientUtil.
``` 

If you get rid of the unused imports, it should work. You can try it locally with `mvn clean package`